pkg/vdr/peer/creator.go

```diff
@@ -59,19 +57,10 @@ func (v *VDR) Create(keyManager kms.KeyManager, didDoc *did.Doc,
 }
 
 //nolint: funlen,gocyclo
-func build(keyManager kms.KeyManager, didDoc *did.Doc,
+func build(didDoc *did.Doc,
 	docOpts *vdrapi.DIDMethodOpts) (*did.DocResolution, error) {
 	if len(didDoc.VerificationMethod) == 0 {
-		id, pubKeyBytes, err := keyManager.CreateAndExportPubKeyBytes(kms.ED25519Type)
-		if err != nil {
-			return nil, fmt.Errorf("failed to create and export public key: %w", err)
-		}
-
-		didDoc.VerificationMethod = append(didDoc.VerificationMethod, did.VerificationMethod{
-			ID:    "#" + id,
-			Type:  ed25519VerificationKey2018,
-			Value: pubKeyBytes,
-		})
+		return nil, fmt.Errorf("verification method is empty")
```

Comment:
Note verificationMethod is optional in generic DID documents: https://w3c.github.io/did-core/#verification-methods

(DID documents may inline all keys in the other sections).